CHANGELOG.md

```diff
@@ -2,66 +2,59 @@
 
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## FAQ
+### What is the purpose of the changelog?
+The purpose of the changelog is for the contributors and maintainers to
+incrementally build the release notes throughout the development process to
+avoid a painful and error-prone process of attempting to compile the release
+notes at release time. On each release the "unreleased" entries of the
+changelog are moved to the appropriate release notes document in the
+`./release-notes` folder.
+
+### Which changes require a CHANGELOG entry?
+Changelogs are intended for _users_. In short, any change that a user of
+OpenSearch might want to be aware of should be included in the changelog. The
+following are some examples of changes that should be in the changelog:
+
+ - A newly added feature
+ - A fix for a user-facing bug
+ - Dependency updates
+ - Fixes for security issues
+
+The following are some examples where a changelog entry  is not necessary:
+
+ - Adding, modifying, or fixing tests
+ - An incremental PR for a larger feature (such features should include _one_
+   changelog entry for the feature)
+ - Documentation or refactoring changes
+ - Build-related changes
+
+Any PR that does not include a changelog entry will result in a failure of the
+validation workflow in GitHub. If the contributor and maintainers agree that no
+changelog entry is required, then the `skip-changelog` label can be applied to
+the PR which will result in the workflow passing.
+
+### Where should I put my CHANGELOG entry?
+Please review the [branching strategy](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#opensearch-branching)
+document. The changelog on the `main` branch will contain sections for the _next
+major_ and _next minor_ releases. Your entry should go into the section it is
+intended to be released in. In practice, most changes to `main` will be backported
+to the next minor release so most entries will likely be in that section.
```

Comment:
re: backport behaves differently

I'd probably not call this out. In general, I suspect this would be modeled as two separate PRs anyway, i.e. "introduce option x" is done first and is backported, then a separate change ("make x the default behavior") is made and not backported.